### PR TITLE
Session2 semantic markup

### PR DESCRIPTION
**Pull request by joordiii**
Leadtech home
Form exercise

### CODE REVIEW

File: staff/jclotet/session2/leaddtech-site-semantic-markup/index.html [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+        </blockquote>
+        <p>Constantly expanding our horizons and going above and beyond contemporary standards, here at leadtech we aim to use the opportunities the 
+          great World Wide Web has given us and continue connecting, communicating and building the best experience possible for our users.</p>
+      </article>
+    </div>
+  </section>
+
+  <section>
+    <h2>what we do</h2>
+    <div>
```
Comment (juanmaguitar) [inline staff/jclotet/session2/leaddtech-site-semantic-markup/index.html]: You don't need a `div` here. The `ul` can be served as a container

File: staff/jclotet/session2/leaddtech-site-semantic-markup/index.html [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+        crunchers, GOT fanatics, polyglots, movers and shakers, decision makers…
+        maybe we’re getting carried away but you get the point.
+        
+        True diversity at its finest!
+      </p>
+      <img src="">
+    </div>
+    <div>
+      <p>We are hiring</p>
+      <a href="">See open positions on LinkedIn</a>
```
Comment (juanmaguitar) [inline staff/jclotet/session2/leaddtech-site-semantic-markup/index.html]: An `a` should always be inside a paragraph.

File: staff/jclotet/session2/leaddtech-site-semantic-markup/index.html [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+          <p>leadtech is taking investors into the future with daily successes, rapid
+              growth and consistently fruitful developments.
+          </p>
+          <a href="">Let's see your strategy</a>
+        </div>
+      </div>
+    </div>
+  </section>
+  <aside>
+    <div>
```
Comment (juanmaguitar) [inline staff/jclotet/session2/leaddtech-site-semantic-markup/index.html]: You don't need this `div`

**Comment by juanmaguitar** [comment]
The form exercise is perfect. Congrats!

**Comment by joordiii** [comment]
Thanks for your comments. I'll take it into account!

**Pull request merged**